libafl_bolts/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libafl_bolts"
-version.workspace = true
```

Comment:
Does that not work?

Reply:
it works but isn't it better to just specify 0.13.1?

Reply:
Dunno, I guess it might be easiest if all versions including dependency versions are specified in the workspace(?)
Either way is ok I guess.